mdtraj/geometry/contact.py

```diff
@@ -151,8 +153,9 @@ def compute_contacts(traj, contacts='all', scheme='closest-heavy', ignore_nonpro
     # now the bulk of the function. This will calculate atom distances and then
     # re-work them in the required scheme to get residue distances
     scheme = scheme.lower()
-    if scheme not in ['ca', 'closest', 'closest-heavy', 'sidechain', 'sidechain-heavy']:
-        raise ValueError('scheme must be one of [ca, closest, closest-heavy, sidechain, sidechain-heavy]')
+
+    if scheme not in ['ca', 'closest', 'closest-heavy', 'sidechain', 'sidechain-heavy', "backbone"]:
```

Comment:
Probably good to use single quotes here for `backbone` to keep it consistent with the rest of the options (looks like double quotes used)